### PR TITLE
doc: improve coverage CLI help text

### PR DESCRIPTION
**Pull request by cknight**
See https://github.com/denoland/manual/issues/42. 

This PR updates the CLI help test for code coverage to make clear the coverage value.

### CODE REVIEW

File: cli/flags.rs [hunk 1]
```diff
@@ -1090,9 +1090,10 @@ fn test_subcommand<'a, 'b>() -> App<'a, 'b> {
         .long("coverage")
         .require_equals(true)
         .takes_value(true)
+        .value_name("coverage_dir")
         .conflicts_with("inspect")
         .conflicts_with("inspect-brk")
-        .help("UNSTABLE: Collect coverage profile data"),
+        .help("Collect coverage profile data into coverage_dir"),
```
Comment (caspervonb) [inline cli/flags.rs:1096]: I have no intention of marking this as stable at this time.

```suggestion
        .help("UNSTABLE: Collect coverage profile data into DIR"),
```

File: cli/flags.rs [hunk 2]
```diff
@@ -1090,9 +1090,10 @@ fn test_subcommand<'a, 'b>() -> App<'a, 'b> {
         .long("coverage")
         .require_equals(true)
         .takes_value(true)
+        .value_name("coverage_dir")
```
Comment (caspervonb) [inline cli/flags.rs:1093]:
```suggestion
        .value_name("DIR")
```

**Review by caspervonb — CHANGES_REQUESTED** [review]

File: cli/flags.rs (hunk 1 above)
Comment (cknight) [reply cli/flags.rs:1096]: Ah right, OK.  I am perhaps confused then as the unstable flag is not required to run the coverage.  I'll add this back.

**Review by caspervonb — APPROVED** [review]
LGTM.

**Review by bartlomieju — APPROVED** [review]
LGTM, thanks

**Pull request merged**